packages/status-react/src/components/main-sidebar/components/get-started/throwaway-profile-found-dialog.tsx

```diff
@@ -1,55 +1,49 @@
 import React from 'react'
 
-import { useAccount } from '../../../../protocol'
 import {
   Avatar,
   Dialog,
-  EmojiHash,
+  // EmojiHash,
+  EthAddress,
   Flex,
   Heading,
   Text,
 } from '../../../../system'
 
+import type { Account } from '../../../../protocol'
+
 interface Props {
-  onSkip: () => void
+  account: Account
 }
 
 export const ThrowawayProfileFoundDialog = (props: Props) => {
-  const { onSkip } = props
-
-  const { account } = useAccount()
+  const { account } = props
 
-  const handleLoadThrowawayProfile = () => {
+  const handleLoad = () => {
     // TODO: load throwaway profile
   }
 
-  if (!account) {
-    return null
-  }
-
   return (
     <Dialog title="Throwaway Profile Found">
       <Dialog.Body gap="5">
         <Flex direction="column" align="center" gap="2">
-          <Avatar size={64} />
+          <Avatar size={64} name={account.username} />
           <Heading weight="600">{account.username}</Heading>
-          <Text color="gray">
-            Chatkey: 0x63FaC9201494f0bd17B9892B9fae4d52fe3BD377
-          </Text>
-          <EmojiHash />
+
+          <EthAddress color="gray">{account.chatKey}</EthAddress>
+          {/* <Text color="gray">Chatkey: {account.chatKey}</Text> */}
+          {/* <EmojiHash /> */}
         </Flex>
         <Text>
-          Throwaway profile is found in your local {"browser's"} storage.
+          Throwaway profile is found in your local storage.
           <br />
-          Would you like to load it and use it?
+          Would you like to use it?
         </Text>
       </Dialog.Body>
 
       <Dialog.Actions>
-        <Dialog.Action variant="outline" onClick={onSkip}>
-          Skip
-        </Dialog.Action>
-        <Dialog.Action onClick={handleLoadThrowawayProfile}>
+        <Dialog.Cancel variant="outline">Skip</Dialog.Cancel>
```

Comment:
It should probably be deleted...

Reply:
After going through the designs, I now think it's there because of the other way a user should be able to login.

For example, once creating a throwaway profile and later using a wallet to create a permanent one. Even though, I'd probably wouldn't got back to being anon, but some might want that option. In which case, it might be worth adding a delete button and a delete option to other flows for creating/obtaining profiles too.

Reply:
Alright, let's delete it.